src/main/scala/org/broadinstitute/dsde/firecloud/dataaccess/HttpGoogleServicesDAO.scala

```diff
@@ -29,19 +29,30 @@ import scala.util.Try
 case class GooglePriceList(prices: GooglePrices, version: String, updated: String)
 
 /** Partial price list. Attributes can be added as needed to import prices for more products. */
-case class GooglePrices(cpBigstoreStorage: UsPriceItem)
+case class GooglePrices(cpBigstoreStorage: UsPriceItem,
+                        cpComputeengineInternetEgressNA: UsTieredPriceItem)
 
 /** Price item containing only US currency. */
 case class UsPriceItem(us: BigDecimal)
 
+/** Tiered price item containing only US currency. */
+case class UsTieredPriceItem(tiers: Map[Long, BigDecimal])
+
 object GooglePriceListJsonProtocol extends DefaultJsonProtocol {
   implicit val UsPriceItemFormat = jsonFormat1(UsPriceItem)
-  implicit val GooglePricesFormat = jsonFormat(GooglePrices, "CP-BIGSTORE-STORAGE")
+  implicit object UsTieredPriceItemFormat extends RootJsonFormat[UsTieredPriceItem] {
```

Comment:
I feel like there has to be a nicer way to do this conversion but I'm not sure how...